sql/Playbook.txt

```diff
@@ -22,10 +22,28 @@ Verify the function was created successfully.
 ** BENEFICIARY_MONTHLY Update Trigger                          **         
 *****************************************************************
 psql -h 127.0.0.1 -U bfd -d fihr -a -f sql/ccw_load_update_trigger.sql 
+psql -h 127.0.0.1 -U bfd -d fihr -a -f sql/ccw_load_delete_trigger.sql 
+psql -h 127.0.0.1 -U bfd -d fihr -a -f sql/ccw_load_insert_trigger.sql 
```

Comment:
A general note about files & folders: Do all of these need to be in separate files? If not, it feels like it adds some minor complexity & risk to the whole thing (e.g. someone forgets to add a trigger); I'd combine them, where possible.

Reply:
Will put into minimum number of files; originally only had support for the update trigger and the insert,delete were added per orig PR review. Not a big deal to coerce into one.